mobile-widgets/qmlmanager.cpp

```diff
@@ -32,6 +32,9 @@
 #include "qt-models/tankinfomodel.h"
 #include "core/downloadfromdcthread.h"
 
+#define STARTUP_TIMER
```

Comment:
i think the logger code needs a better implementation - i.e. a better implementation of a common C/C++ logging pattern controlled by a pre-processor macro.
yet i don't have the time to propose a better solution ATM.

Reply:
Please, I am not rewriting our logger or anything, just providing a simple test tool for a specific problem.

Might be better if I just close this, and do my own testing....it is a pity because I think we could benefit from having numbers from different devices, but I do not have the time to make a new elegant logger solution.